common/source/docs/common-blheli32-passthru.rst

```diff
@@ -1,62 +1,149 @@
 .. _common-blheli32-passthru:
 
-BLHeli32/ BLHeli-S Pass-Through Support
-=======================================
+BLHeli32 and BLHeli_S ESCs
+==========================
 
-In addition to offering enhanced capabilities such as reversing and often, telemetry, BLHeli capable ESCs have the ability to have their parameters and features programmed through the autopilot while it is connected to a PC (if the autopilot firmware provides this capability also). This BLHeli pass-through protocol allows you to configure and upgrade your ESCs without having to disconnect them from your vehicle. You can plug a USB cable into your autopilot and run the BLHeliSuite or BLHeliSuite32 software for Windows to configure your ESCs. 
+The BLHeli protocol is built on the :ref:`DShot <common-dshot-escs>` protocol and offers these additional features
```

Comment:
I'm not sure I agree actually.. maybe we should remove the word "protocol" because I think BLHeli is ESC software and it's accompanying configuration application.

I don't think BLHeli itself actually allows you to select which protocol the ESCs will use (PWM, OneShot, DShot).  I agree it does allow configuration of timing and motor direction.

Reply:
we are both wrong...you can select to disable the PWM /Oneshot 125 compatibility in a BLHeli_S ESC...but evidentially not a BLHeli32 ESC..there is not configuration option for it......but I dont think its a big enough deal to even mention....so perhaps this: 
```suggestion
The BLHeli firmware and configuration applications were developed to allow the configuration of ESCs and provide additional features. ESCs with this firmware allow parameters such as timing, motor direction, LEDs, motor drive frequency, etc. to be configured. BLHeli firmware  has many variations, such as BLHeli, BLHeli_S, BLHeli32, etc. 
```

Reply:
ok thanks.  changed to something very close to the suggestion.